openarena/Program.cs

```diff
@@ -69,17 +76,17 @@ public static void DataReceived(object sender, DataReceivedEventArgs e)
                     LogMessage("Server is getting terminated.");
                 }
             }
-            else if (e.Data.Contains("ClientBegin:"))
+            else if (e.Data.Contains("ClientBegin:")) // new player connected
             {
-                players.Add(new ConnectedPlayer("gamer"+ new Random().Next(0,21)));
+                players.Add(new ConnectedPlayer("gamer" + new Random().Next(0,21)));
                 GameserverSDK.UpdateConnectedPlayers(players);
             }
-            else if (e.Data.Contains("ClientDisconnect:"))
+            else if (e.Data.Contains("ClientDisconnect:")) // player disconnected
             {
                 players.RemoveAt(new Random().Next(0, players.Count));
```

Comment:
Minor: not related to your document change, does it make sense to terminate the game here if players.Count == 0.